File: Wrap/ Wrap JavaScript/Wrap7.js
const scanner = require("prompt-sync")();
const number = scanner("Enter a number:");
         
    for(let count = 1; count <= number; count++){ 

        let sum = 0;

       for(let counter = 1; counter < count; counter++){

         if(count % counter == 0){

            sum += counter;

        }
        }
        
         if (sum == count){

         console.log("perfect number :" + count);
    
        }
    }
    

File: Wrap/Wrap java/Wrap7.java
import java.util.Scanner;
public class Wrap7{

    public static void main(String[] args){

    Scanner input = new Scanner(System.in);

    System.out.println("Enter a number");
    int number = input.nextInt();
    
    
     
    for(int count = 1; count <= number; count++){ 

        int sum = 0;

       for(int counter = 1; counter < count; counter++){

         if(count % counter == 0){

            sum += counter;

        }
        }
        
         if (sum == count){

         System.out.println("perfect number :" + count);
    
        }
    }
    
    }
}

